### ISSUE TITLE
Filtering doesn't respond to hub being shut down

### ISSUE THREAD

**Issue by maryjgoldman** [acthp/ucsc-xena-client#37]
Filtering by dataset in hub that's no longer there doesn't update when the hub is turned off and the refresh button clicked.
To reproduce:
1. Start up your local hub with a dataset in a cohort that is also on the main UCSC hub
2. Add a couple random columns of data to the view
3. Filter by dataset from your local hub and observe the change in the view
4. Shut down the hub and click the new refresh button
--> Observe that data still filtered by dataset from local hub that isn't there

**Comment by achang28** [comment]
@maryjgoldman , just to confirm the workflow, would there be a 4a (go to "Data Hubs"), and 4b (unselected UCSC hub, and select TCGA Hub)?

**Comment by maryjgoldman** [comment]
No. You do not need to go to the Data Hubs page after you shut down your local hub. If you local hub is not running, we do not show data from it, even if the user has tried to select it.

I'm confused for you 4b. The TCGA hub has nothing to do with this. Ping me?
